python-sdk/tests/benchmark/infrastructure/kubernetes/job.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: benchmark-
+  name: benchmark
```

Comment:
@pankajastro I think we will need a unique name since we can have multiple runs concurrently. 
Maybe we can do something like - https://github.com/astronomer/astro-sdk/pull/927

Reply:
ok, I'll wait for https://github.com/astronomer/astro-sdk/pull/927 to merge and reuse your changes i.e tag itself WDYT?

Reply:
@pankajastro I think we can copy the code and merge/resolve conflicts later, in the interest of time.

Reply:
I can see there is still some discussion going on in your PR.  The teardown is also problematic I feel we should take cluster name from env that way it would be completely isolated and even teardown will be fine I have created a ticket https://github.com/astronomer/astro-sdk/issues/934  WDYT?